tests/integ/link_test.py

```diff
@@ -1115,7 +1259,7 @@ def testPostLinkMultiple(self):
         obj_links = rspJson["links"]
         self.assertEqual(len(obj_links), 6)
         expected_group_links = ("g1", "g2", "g1.1", "g1.2", "g1.2.1", )
-        expected_dset_links = ("dset1.2", "dset2.2", "dset1.1.1", "dset1.1.2", "dset2.1", )
```

Comment:
Was there a reason to remove `dset1.2` from this test? It doesn't cause any issues when added back in.